beams/bin/service_main.py

```diff
@@ -1,25 +1,32 @@
-# toss arg parse here to start
 import logging
 import time
 from multiprocessing.managers import BaseManager
+from typing import Callable, Dict
 
 from beams.logging import setup_logging
 from beams.service.helpers.worker import Worker
 from beams.service.remote_calls.command_pb2 import CommandMessage, CommandType
-from beams.service.rpc_handler import RPCHandler
+from beams.service.rpc_handler import (RPCHandler, TreeIdKey,
+                                       get_tree_from_treetickerdict)
 from beams.service.tree_ticker import TreeState, TreeTicker
 
 logger = logging.getLogger(__name__)
 
+TreeTickerDict = Dict[TreeIdKey, TreeTicker]
+
 
 class BeamsService(Worker):
     def __init__(self):
         # TODO: make a singleton. Make process safe by leaving artifact file
         super().__init__("BeamsService", grace_window_before_terminate_seconds=0.5)
 
-        # remote manager: https://docs.python.org/3/library/multiprocessing.html#using-a-remote-manager
         class SyncMan(BaseManager):
-            pass
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                # For type hinting purposes only
+                self.TreeTicker: type[TreeTicker]
+                self.TreeState: type[TreeState]
+                self.get_tree_dict: Callable[[], TreeTickerDict]
```

Comment:
Question: is there an advantage to type hinting like this in `__init__` as opposed to e.g.
```suggestion
            TreeTicker: type[TreeTicker]
            TreeState: type[TreeState]
            get_tree_dict: Callable[[], TreeTickerDict]
```